### DISCUSSION TITLE
群组功能怎么使用？@没有反应怎么办？

### DISCUSSION THREAD

**Discussion by xhhbdxh** [TBXark/ChatGPT-Telegram-Workers#122] [Q&A] [answered]
如题

**Comment by tbxark** [comment, marked as answer]
- 配置`GROUP_CHAT_BOT_ENABLE `为`true`
- 配置`TELEGRAM_BOT_NAME`, 机器人名字 `xxx_bot`, 顺序必须和TELEGRAM_AVAILABLE_TOKENS 一致, 必须设置否则无法在群聊中使用
- 必须把群ID加到白名单`CHAT_GROUP_WHITE_LIST`才能使用, 否则任何人都可以把你的机器人加到群组中，然后消耗你的配额。
- 受限TG的隐私安全策略，如果你的群组是公开群组或超过2000人，请将机器人设置为管理员，否则机器人无法响应@机器人的聊天消息。
- 必须在`botfather`中设置`/setprivacy`为`Disable`，否则机器人无法响应@机器人的聊天消息。
